docs/ref/utils.txt

```diff
@@ -826,6 +826,8 @@ appropriate entities.
 
     Can be called multiple times on a single string.
 
+    Can also be used as a decorator.
```

Comment:
Need a `.. versionchanged:: 1.10` annotation and a mention in the release notes.